src/EditorFeatures/Core/Preview/AbstractPreviewFactoryService.cs

```diff
@@ -709,6 +711,17 @@ private async ValueTask<IDifferenceViewerPreview<TDifferenceViewer>> CreateNewDi
 
             rightWorkspace?.Dispose();
             rightWorkspace = null;
+
+            // We're closing - change the content type of the buffers we created to inert
+            // as a signal for the editor / LSP client to unhook from them.
+            var inertContentType = _contentTypeRegistryService.GetContentType(StandardContentTypeNames.Inert);
+            buffersToClose = buffersToClose
+                .Add(originalBuffer)
```

Comment:
>originalBuffer [](http://example.com/codeflow?start=21&length=14)

Would it make sense for the caller to specify all the buffers to close? We take in a param that indicates what buffers to close, seems like we should just have the caller specify all the buffers to close in it.